server/graphql/common/expenses.ts

```diff
@@ -519,18 +519,21 @@ const getPayoutMethodFromExpenseData = async (expenseData, remoteUser, fromColle
   if (expenseData.payoutMethod) {
     if (expenseData.payoutMethod.id) {
       const pm = await models.PayoutMethod.findByPk(expenseData.payoutMethod.id);
-      if (!pm || !remoteUser.isAdmin(pm.CollectiveId)) {
-        throw new Error("This payout method does not exist or you don't have the permission to use it");
+      if (!pm) {
+        throw new Error('This payout method does not exist.');
       }
+      // Special case: Payout Method from the Host for "Expense Accross Hosts"
+      // No need for extra checks
       if (
-        // Payout Method from Collective
-        pm.CollectiveId !== fromCollective.id &&
-        // Bank Account or PayPal Payout Method from Host
-        !(
-          pm.CollectiveId === fromCollective.HostCollectiveId &&
-          [PayoutMethodTypes.BANK_ACCOUNT, PayoutMethodTypes.PAYPAL].includes(pm.type)
-        )
+        pm.CollectiveId === fromCollective.HostCollectiveId &&
```

Comment:
So conceptually, we'll always allow collective admins to use their host's payout methods - not only for cross-hosts, since this condition will be valid for an expense from Babel to Webpack. I don't see how this could be directly abused, it looks pretty safe.

Reply:
It's not really desirable in that case. But it's safe and the Frontend should not expose this option at this stage.